pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "detection_rules"
-version = "1.4.1"
+version = "1.4.2"
```

Comment:
```suggestion
version = "1.4.3"
```

Reply:
Given that this will merge First - https://github.com/elastic/detection-rules/pull/5115#pullrequestreview-3227461688